config/store.go

```diff
@@ -26,9 +26,6 @@ type Store interface {
 	// Load updates the current configuration from the backing store, possibly initializing.
 	Load() (err error)
 
-	// Save writes the current configuration to the backing store.
```

Comment:
Can we now also eliminate the `Save` implementations on the concrete store instances?

Reply:
Also, can you update the comment on `Set` above in this interface to reflect the new functionality?

Reply:
Yes, we can eliminate all `Save` implementations, and I updated the comment on `Set`.